operations/host_spawn.go

```diff
@@ -480,6 +480,58 @@ func hostStart() cli.Command {
 	}
 }
 
+func hostSSH() cli.Command {
+	const (
+		identityFlagName = "identity"
```

Comment:
should this be called `identity_file`?

Reply:
I think that's long to type out.

Reply:
Ah but I see that is the ssh name :( So i guess yes haha